zerver/lib/url_preview/oembed/__init__.py

```diff
@@ -0,0 +1,14 @@
+from typing import Optional, Any
+from six import text_type
+from pyoembed import oEmbed, PyOembedException
+
+
+def get_oembed_data(url, maxwidth=640, maxheight=480):
```

Comment:
Yeah, I understand we want to specify something here, the question was what your reasoning was for picking 640x480 rather than e.g. 320x240 or something else.